openedx/core/djangoapps/plugins/constants.py

```diff
@@ -0,0 +1,60 @@
+# Name of the class attribute to put in the AppConfig class of the Plugin App.
+PLUGIN_APP_CLASS_ATTRIBUTE_NAME = u'plugin_app'
+
+
+# Name of the function that belongs in the plugin Django app's settings file.
+# The function should be defined as:
+#     def plugin_settings(settings):
+#         # enter code that should be injected into the given settings module.
+PLUGIN_APP_SETTINGS_FUNC_NAME = u'plugin_settings'
+
+
+class ProjectType(object):
```

Comment:
Weren't we going to use the `enum34` backport to make these enums?

Reply:
I need to be able to call `getattr` with these enum values.  Does the new `Enum` class have a way to get the string value of the enum field?

Reply:
Looks like it: https://docs.python.org/3/library/enum.html#programmatic-access-to-enumeration-members-and-their-attributes

Reply:
Ok.  Not ideal for our case though.  We'll have to remember to call `.value` whenever we pass the `Enum` as a parameter to a `dict`'s `get` or to a `getattr`.  I'm not sure converting the classes to Enums actually helps much in our case where the string values are more significant than the constant names.  I'll take a stab at this and see the impact on the code.

Reply:
Ok.  I took a stab at changing these to be `Enum`s.  However, the usage of them no longer works as designed. 
In particular, Plugin apps need to either always use their string values or always use their `Enum` values as `dict` keys.  Either that, or the plugins code needs to check for both (string and Enum) when looking up in the `dict`.  Neither is ideal and I'd rather keep these as `Class`es given these issues.